pkg/cmd/operator/analyze.go

```diff
@@ -104,9 +112,64 @@ func (o *AnalyzeOptions) Complete() error {
 	return nil
 }
 
+func (o *AnalyzeOptions) IsArchiveMode() bool {
+	return len(o.ArchivePath) != 0
+}
+
+func (o *AnalyzeOptions) IsKubeconfigMode() bool {
+	return len(o.Kubeconfig) != 0
+}
+
+func (o *AnalyzeOptions) makeMustGatherDataSource(ctx context.Context, cmd *cobra.Command) (analyze.DataSource, error) {
+	// TODO
+	return nil, errors.ErrUnsupported
+}
+
+func (o *AnalyzeOptions) makeKubeconfigDataSource(ctx context.Context, cmd *cobra.Command) (analyze.DataSource, error) {
+	client, err := kubernetes.NewForConfig(o.RestConfig)
+	if err != nil {
+		return nil, fmt.Errorf("can't create kubernetes client: %w", err)
+	}
+	scyllaClient, err := scyllaversioned.NewForConfig(o.RestConfig)
+
+	kubeInformerFactory := informers.NewSharedInformerFactory(client, durationInfinity)
+	scyllaInformers := scyllainformers.NewSharedInformerFactory(scyllaClient, durationInfinity)
+
+	var ds = analyze.NewDataSource(append(
+		[]any{},
+		kubeInformerFactory.Core().V1().Services().Lister(),
+		kubeInformerFactory.Core().V1().Secrets().Lister(),
+		kubeInformerFactory.Core().V1().ConfigMaps().Lister(),
+		kubeInformerFactory.Core().V1().ServiceAccounts().Lister(),
+		scyllaInformers.Scylla().V1().ScyllaClusters().Lister(),
+	))
+
+	kubeInformerFactory.Start(ctx.Done())
```

Comment:
This is blocking, you have to start it in separate goroutine and synchronize somewhere

Reply:
done